packages/cli/src/commands/proxy.ts

```diff
@@ -0,0 +1,41 @@
+import * as signale from 'signale';
+import { CommandModule } from 'yargs';
+import { createMultiProcessPrism, CreatePrismOptions, createSingleProcessPrism } from '../util/createServer';
+import getHttpOperations from '../util/getHttpOperations';
+import options from './options';
+
+const proxyCommand: CommandModule = {
+  describe: 'Start a proxy server with the given spec file',
```

Comment:
Let's not talk about spec files, are are trying to stick to a unwritten styleguide but we call them "description documents". So "document" is fine.